package.json

```diff
@@ -4,7 +4,7 @@
   "description": "alpha team project",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=test nyc --reporter=text --reporter=lcov mocha  --require ts-node/register   test/server-run.ts --coverage",
+    "test": "cross-env NODE_ENV=test nyc --reporter=text --reporter=lcov mocha  --require ts-node/register   test/*.ts --coverage",
```

Comment:
this test will not run you must first do demigrate and migrate then run test

Reply:
done